src/include/lean/lean.h

```diff
@@ -991,6 +991,8 @@ static inline size_t lean_string_byte_size(lean_object * o) { return sizeof(lean
 /* instance : inhabited char := ⟨'A'⟩ */
 static inline uint32_t lean_char_default_value() { return 'A'; }
 LEAN_EXPORT lean_obj_res lean_mk_string_from_bytes(char const * s, size_t sz);
+LEAN_EXPORT lean_obj_res lean_mk_ascii_string(char const * s);
```

Comment:
Are we using this in any performance-critical spot?

Reply:
I introduced that function for Nat.repr